app/pages/solutions-list/filterType/foundation/ui.test.js

```diff
@@ -42,6 +42,20 @@ test('should display the page description', async (t) => {
     .expect(await extractInnerText(pageDescription)).eql('These Catalogue Solutions meet the 6 Foundation Capabilities that are the minimum requirement to enable a GP practice to operate.');
 });
 
+test('should not display the compare description', async (t) => {
```

Comment:
we don't show it on the foundation results page?

Reply:
no - clarified with Josh on the story

Reply:
👍